mobile-widgets/qml/StartPage.qml

```diff
@@ -32,12 +26,32 @@ Kirigami.ScrollablePage {
 			text: manager.startPageText
 			wrapMode: Text.WordWrap
 		}
-		CloudCredentials {
-			id: cloudCredentials
+		Kirigami.Label {
+			id: explanationTextBasic
+			visible: !showPin
 			Layout.fillWidth: true
 			Layout.margins: Kirigami.Units.gridUnit
-			Layout.topMargin: 0
-			property int headingLevel: 3
+			Layout.topMargin: Kirigami.Units.gridUnit * 3
+			text: qsTr("To use Subsurface-mobile with Subsurface cloud storage, please enter your cloud credentials.\n\n") +
+				qsTr("If this is the first time you use Subsurface cloud storage, enter a valid email (all lower case) " +
+				"and a password of your choice (letters and numbers).\n\n") +
+				qsTr("To use Subsurface-mobile only with local data on this device, select " +
+				"the no cloud buttton above.")
```

Comment:
i wonder if this is going to introduce a number of strings for translation instead of single one.
probably best to concat these with `"str1" + "str2"`
or use a single quoted string:
```
    property string str: qsTr("test
new line
new line")
```